guides/common/modules/proc_synchronizing-a-custom-repository.adoc

```diff
@@ -0,0 +1,26 @@
+[id="Synchronizing_a_Custom_Repository_{context}"]
+= Synchronizing a Custom Repository
+
+Use this procedure to synchronize content from a custom repository on a connected {ProjectServer} and a disconnected {ProjectServer} via {ISS} (ISS) Network Sync.
+
+Follow the procedure for the connected {ProjectServer} before completing the procedure for the disconnected {ProjectServer}.
+
+.Connected {ProjectServer}
+. In the {ProjectWebUI}, navigate to *Content* > *Products*.
+. Click on the custom product.
+. Click on the custom repository.
+. Copy the *Published At:* URL.
+. Continue with the procedure on disconnected {ProjectServer}.
+
+.Disconnected {ProjectServer}
+. In the {ProjectWebUI}, navigate to *Content* > *Products*.
+. Create your custom product with the following:
+* Upstream URL: Paste the link that you copied earlier.
+* On the connected {ProjectServer}, download the file `https:///pub/katello-server-ca.crt` using `curl` or `wget`.
+* Transport the file to the disconnected server using any method.
+* On the disconnected {ProjectServer}, create an SSL Content Credential with the contents of this file.
```

Comment:
Just to clarify, this talks about the below procedure @jeremylenz?
https://access.redhat.com/documentation/en-us/red_hat_satellite/6.12/html/managing_content/importing_content_content-management#Importing_Custom_SSL_Certificates_content-management